### ISSUE TITLE
Sheila doesn't power up

### ISSUE THREAD

**Issue by amcewen** [DoESLiverpool/somebody-should#1656]
Although she now [passes her PAT test](https://github.com/DoESLiverpool/somebody-should/issues/1641), Sheila doesn't power up.

A *very* quick bit of debugging shows that there's mains getting to the low-voltage power supply, but there seems to be nothing coming out on the 24V or the 5V sides.  So maybe the work for the PAT test has killed that power supply?  Needs more investigation.

**Comment by Sean-anotherone** [comment]
Update from today - Adrian's quick debug confirmed, so removed the low voltage supply and replaced it with separate 5v and 24v supplies (these are not securely fitted atm)  Machine still does not power up, however, the 24v supply shows a fault LED.

Next steps :

1. Bench test the original supply ( I did not do this, but given the above it might have simply shut down because of a downstream fault) and refit it if it bench tests good.
2. Disconnect the 24v loads one at a time until the machine does power up - current information points  to a non-psu fault, but not definitively.  24v loads are the stepper drivers and ?? what else ??


Queries that arise - did this happen at PAT testing?  ( I ask because a PAT test blew the PSU on a £300k electron microscope I ran in a past life), if yes, did Gerald fail after PAT test?  Is PAT tester killing equipment?  Just speculating based on being burned once long ago.

Here's the shot of the supply connections I took 
Note "unexpected" assignment of the +ve and -ve colours
![image](https://user-images.githubusercontent.com/39928682/153512648-401480c6-cee4-4eb7-9fa2-7ab7a00ec8b4.png)

**Comment by amcewen** [comment]
Sheila was working before the PAT testing, so that or the filter replacement could be what's killed the power supply :disappointed: 

Gerald hasn't been PAT tested yet, so his failure isn't related to PAT testing.

**Comment by magman2112** [comment]
With some further testing, I found that the DC connections were actually reversed (checked the connections on the controller and the Stepper drivers). On connecting them the right way, the laser powered up with the replacement PSU's that @Sean-anotherone installed on Thursday.

Further testing found that the original PSU worked when connected in the right way, so I re-installed this and tested and the laser cuts fine.

**Comment by Sean-anotherone** [comment]
I am so glad I took a photo of the way I found it when I took a look at the machine last Thursday !!!
Power supply connected the wrong way round is right up there with the on-off switch being set to the wrong position...
I'm not going to suggest that I've never ever done this myself, but not me on this occasion.
Sean





    On Saturday, 12 February 2022, 17:51:21 GMT, Mike Gorman ***@***.***> wrote:  
 
 


With some further testing, I found that the DC connections were actually reversed (checked the connections on the controller and the Stepper drivers). On connecting them the right way, the laser powered up with the replacement PSU's that @Sean-anotherone installed on Thursday.

Further testing found that the original PSU worked when connected in the right way, so I re-installed this and tested and the laser cuts fine.

—
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.
